operations/preinstall/Change_River_BMC_Credentials.md

```diff
@@ -0,0 +1,136 @@
+
+  ```bash
+  $ curl -s -k -u root:<DFLTPW> https://<BLADENAME_OR_IP>/redfish/v1/AccountSystem/Accounts | jq
+  {
+    "@odata.context": "/redfish/v1/$metadata#ManagerAccountCollection.ManagerAccountCollection",
+    "@odata.id": "/redfish/v1/AccountService/Accounts",
+    "@odata.type": "#ManagerAccountCollection.ManagerAccountCollection",
+    "Name": "Accounts Collection",
+    "Description": "BMC User Accounts",
+    "Members@odata.count": 4,
+    "Members": [
+      {
+        "@odata.id": "/redfish/v1/AccountService/Accounts/1"
+      },
+      {
+        "@odata.id": "/redfish/v1/AccountService/Accounts/2"
+      },
+      {
+        "@odata.id": "/redfish/v1/AccountService/Accounts/3"
+      },
+      {
+        "@odata.id": "/redfish/v1/AccountService/Accounts/4"
+      }
+    ]
+  }
+```
+
+3. For each account listed, use *curl* to find the one which describes the *root* account ("UserName": "root").  
+
+**NOTES:**
+ - The root account can be any of the listed accounts -- no guarantees as to which one it will be.
+ - If the account information contains an *etag* entry, note this number, as it will be required when setting the password.
+
+  ```bash
+  $ curl -s -k -u root:<DFLTPW> https://<BLADENAME_OR_IP>/redfish/v1/AccountSystem/Accounts/1 | jq
+  {
+    "@odata.context": "/redfish/v1/$metadata#ManagerAccount.ManagerAccount",
+    "@odata.id": "/redfish/v1/AccountService/Accounts/1",
+    "@odata.type": "#ManagerAccount.v1_1_1.ManagerAccount",
+    "@odata.etag": "W/\"570254F2\"",
+    "Id": "3",
+    "Name": "User Account",
+    "Description": "User Account",
+    "Enabled": true,
+    "Password": null,
+    "UserName": "root",
+    "RoleId": "NoAccess",
+    "Links": {
+      "Role": {
+        "@odata.id": "/redfish/v1/AccountService/Roles/NoAccess"
+      }
+    }
+  }
+  ```
+
+4. Set the new password for this account.  Use the ETAG in the header if needed.
+
+  ```bash
+  $ curl -s -k -u root:<DFLTPW> -H "If-None-Match: 570254F2" -H "Content-Type: application/json" -X PATCH -d '{"Password":"<NEWPW>"}' https://<BLADENAME_OR_IP>/redfish/v1/AccountSystem/Accounts/1
+  $
+  ```
+
+5. Test to be sure the new password works.  If the password set operation didn't work, then this will fail.
+
+  ```bash
+  $ curl -s -k -u root:<NEWPW> https://<BLADENAME_OR_IP>/redfish/v1/AccountSystem
+  {
+    "@odata.context": "/redfish/v1/$metadata#ManagerAccount.ManagerAccount",
+    "@odata.id": "/redfish/v1/AccountService/Accounts/1",
+    "@odata.type": "#ManagerAccount.v1_1_1.ManagerAccount",
+    "@odata.etag": "W/\"570254F2\"",
+    "Id": "3",
+    "Name": "User Account",
+    "Description": "User Account",
+    "Enabled": true,
+    "Password": null,
+    "UserName": "root",
+    "RoleId": "NoAccess",
+    "Links": {
+      "Role": {
+        "@odata.id": "/redfish/v1/AccountService/Roles/NoAccess"
+      }
+    }
+  }
+  ```
+
+### Set default credentials for high speed network switch BMCs
```

Comment:
Maybe we should add an explicit wording stating that the root user creds for both liquid cooled and air cooled switch BMCs need to be identical.

Reply:
Done.